app/models/task.py

```diff
@@ -2,4 +2,39 @@
 
 
 class Task(db.Model):
-    task_id = db.Column(db.Integer, primary_key=True)
+    id = db.Column(db.Integer, primary_key=True)
+    title = db.Column(db.String)
+    description = db.Column(db.String)
+    completed_at = db.Column(db.DateTime, default=None)
+    goal = db.relationship("Goal", back_populates="tasks")
+    goal_id_parent = db.Column(db.Integer, db.ForeignKey("goal.goal_id"))
+
+    @classmethod
+    def from_dict(cls, data_dict, goal=None):
+        return cls(
+            title=data_dict["title"],
+            description=data_dict["description"],
+            goal=goal
+        )
+
+    def to_dict(self):
+        dictionary={}
```

Comment:
Since the `if` and `else` statements both declare the same variable name, we don't need to declare it globally first. That's because no matter what option is chosen, the variable will be declared and assigned. We can get rid of line 21 if we want to!